scripts/dts/python-devicetree/src/devicetree/dtlib.py

```diff
@@ -731,6 +733,16 @@ def __init__(self, filename, include_path=()):
         self._remove_unreferenced()
         self._register_labels()
 
+    @property
+    def root(self) -> Node:
+        """
+        See the class documentation.
+        """
+        # This is necessary because mypy can't tell that we never
+        # treat self._root as a non-None value until it's initialized
+        # properly in _parse_dt().
+        return self._root       # type: ignore
```

Comment:
another option would be to assert for `self._root`, but that's fine

Reply:
> another option would be to assert for `self._root`, but that's fine

I haven't done any profiling here, but I really want to avoid any runtime overhead, no matter how small, because this code is run so many times in CI.